piksi_tools/console/settings_view.py

```diff
@@ -212,6 +227,40 @@ def _factory_default_button_fired(self):
                         + "Are you sure you want to reset to factory defaults?"
     confirm_prompt.run(block=False)
 
+  def _settings_save_to_file_fired(self):
```

Comment:
I've seen situations where people needed to change this (and I definitely have several times myself). I kinda feel like it would be more confusing if the option was hidden. People do use the USB port for connection to systems other than just the console it seems.